x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_alerts/alerts_detection_callouts_index_outdated.cy.ts

```diff
@@ -33,7 +33,6 @@ const waitForPageTitleToBeShown = () => {
   cy.get(PAGE_TITLE).should('be.visible');
 };
 
-// TODO: https://github.com/elastic/kibana/issues/161539 Does it need to run in Serverless?
```

Comment:
I didn't see any reason this needs to run in serverless as it was related to the migration of the alerts index.